### PR TITLE
gprof2dot: Add new package

### PR DESCRIPTION
**Pull request by wizeman**
gprof2dot is a Python script to convert the output from many profilers
into a dot graph.

### CODE REVIEW

File: pkgs/development/tools/profiling/gprof2dot/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchgit, pythonPackages }:
+
+pythonPackages.buildPythonPackage {
+  name = "gprof2dot-2015-04-27";
+
+  src = fetchgit {
```
Comment (edwtjo) [inline pkgs/development/tools/profiling/gprof2dot/default.nix]: Please use fetchFromGitHub instead, imho an archive is always better than a repo clone. The specific combo of using both fetchgit and git:// is especially bad for proxied users.

Comment (wizeman) [inline pkgs/development/tools/profiling/gprof2dot/default.nix]: @edwtjo Sounds good, I will change it. Thanks!

**Comment by wizeman** [comment]
Ok, switched to `fetchFromGitHub` as suggested by @edwtjo.

**Comment by edwtjo** [comment]
All good

**Pull request merged**